### PR TITLE
test: add network tests

### PR DESCRIPTION
**Pull request by Letty5411**
Signed-off-by: letty <letty.ll@alibaba-inc.com>

### Ⅰ. Describe what this PR did
Add basic network CLI tests.

### CODE REVIEW

File: test/cli_network_test.go [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+	check.Suite(&PouchNetworkSuite{})
+}
+
+// SetUpSuite does common setup in the beginning of each test suite.
+func (suite *PouchNetworkSuite) SetUpSuite(c *check.C) {
+	SkipIfFalse(c, environment.IsLinux)
+	command.PouchRun("pull", busyboxImage).Assert(c, icmd.Success)
+}
+
+// TestNetworkDefault tests default bridge network is created.
```
Comment (ZouRui89) [inline test/cli_network_test.go]: default bridge network is created. -> the creation of default bridge network.

**Comment by ZouRui89** [comment]
Maybe adding a duplicate network creation test case is better.
image

File: test/cli_network_test.go (hunk 1 above)
Comment (Letty5411) [reply test/cli_network_test.go]: ok~

**Comment by Letty5411** [comment]
@ZouRui89  OK, as the api_network_test.go has a duplicate network name test, I didn't add it in CLI test. But add one is also OK.

**Comment by ZouRui89** [comment]
LGTM

**Comment by allencloud** [comment]
Thanks for your review @ZouRui89  and your work @Letty5411 .
Merging now.

**Pull request merged**